openmetadata-ui/src/main/resources/ui/src/components/Suggestions/SuggestionsProvider/SuggestionsProvider.tsx

```diff
@@ -62,48 +63,51 @@ const SuggestionsProvider = ({ children }: { children?: ReactNode }) => {
   const publish = usePub();
 
   const [loading, setLoading] = useState(false);
-  const [suggestionLimit, setSuggestionLimit] = useState<number>(10);
+  const [suggestionLimit, setSuggestionLimit] = useState<number>(PAGE_SIZE);
```

Comment:
Why we are using state and not using usePaging hook?

Reply:
Here we don't require any Next, Previous, or limit changing mechanism. State is only set to get all the value from total and fetch them single time. On the user action.